lib/desired-caps.js

```diff
@@ -2,138 +2,139 @@
 const PLATFORM_NAME_IOS = 'iOS';
 const PLATFORM_NAME_TVOS = 'tvOS';
 
-const desiredCapConstraints = /** @type {const} */({
-  platformName: { // override
+const desiredCapConstraints = /** @type {const} */ ({
```

Comment:
Please revert autoformatter changes. They make it very hard to distinguish where the "real" difference is

Reply:
Yeah, currently the original files are not aligned with prettier rules but this repository forces developers to format in each commit with husky.
Can I submit a PR to apply the format to all files?

Reply:
Can you take a look? https://github.com/appium/appium-xcuitest-driver/pull/1973